src/clients/c++/request.h

```diff
@@ -329,6 +334,23 @@ class InferContext {
     /// \param input The vector holding tensor string values.
     /// \return Error object indicating success or failure.
     virtual Error SetFromString(const std::vector<std::string>& input) = 0;
+
+    /// Set tensor values for this input by reference into a shared memory
+    /// region. The values are not copied and so the shared memory region and
+    /// its contents must not be modified or destroyed until this input is no
+    /// longer needed (that is until the Run()
+    //// call(s) that use the input have completed). For batched inputs this
+    /// function must be called batch-size times to provide all tensor values
```

Comment:
Not called multiple times for batched input, correct?

Reply:
Right. This should be once per batch.